Framework/Algorithms/test/Stitch1DTest.h

```diff
@@ -134,21 +107,17 @@ class Stitch1DTest : public CxxTest::TestSuite {
 
   Stitch1DTest() {
 
-    e = MantidVec(10, 0);
-    x = MantidVec(11);
-    const double xstart = -1;
-    const double xstep = 0.2;
-    LinearSequence<MantidVec::value_type> sequence(xstart, xstep);
-    std::generate(x.begin(), x.end(), sequence);
-
-    MantidVec y = {0, 0, 0, 3, 3, 3, 3, 3, 3, 3};
+    HistogramX x(11, LinearGenerator(-1, 0.2));
+    HistogramY y(std::vector<double>{0, 0, 0, 3, 3, 3, 3, 3, 3, 3});
```

Comment:
`HistogramY` can be constructed directly from initializer_list, no need to make a detour to `std::vector`.